metrics/plugins/prometheus/README.md

```diff
@@ -0,0 +1,27 @@
+## Prometheus
+
+Plugin to collect metrics and send them to Prometheus server.
+
+### API
+
+Import Prometheus Plugin:
+
+```lua
+local prometheus = require('metrics.plugins.prometheus')
+```
+
+#### `prometheus.collect_http()`
+Returns:
+```lua
+{
+    status = <http-status>,
+    headers = <headers>,
+    body = <body>,
```

Comment:
There is no body format explanation, is it described somewhere in this project or there is some links to format describing?